src/config.h

```diff
@@ -142,7 +142,8 @@ struct Config {
     // will be elided with ".." put at the front. Set to 0 or a negative number
     // to disable eliding.
     int includeMaxPathSize = 30;
-
+    int MaxPreloadedSessions = 10;
```

Comment:
Are these being used anywhere? Consider removing SetMaxPreloadedSessions/SetMaxCompletionSessions and just have clang_complete.cc look at g_config->maxPreloadedSessions/maxCompleteSessions.

Also match current spelling (lower case first letter) and add doc comments